godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml

```diff
@@ -0,0 +1,48 @@
+<package>
+  <name>abb_irb1200_support</name>
+  <version>1.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 1200 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 1200 manipulators. This currently includes the
+      IRB 1200-5/0.9m (1200-5) only.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in the
+      <a href="https://library.e.abb.com/public/ff04442733894d2fa3da90cfaa0d3eee/IRB1200_ROB0275EN_A.pdf">ABB IRB 1200
+      technical data sheet</a>. All urdfs /
+      xacros are based on the default motion and joint velocity limits, unless
+      noted otherwise (ie: no support for high speed joints, extended / limited
+      motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>(Southwest Research Institute)</author>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</maintainer>
+  <license>Apache2</license>
+
+  <url type="website">http://wiki.ros.org/abb_irb6640_support</url>
+  <url type="bugtracker">https://github.com/ros-industrial/abb/issues</url>
+  <url type="repository">https://github.com/ros-industrial/abb</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend version_gte="1.9.55">roslaunch</build_depend>
+
+  <run_depend>abb_driver</run_depend>
+  <run_depend>joint_state_publisher</run_depend>
+  <run_depend>robot_state_publisher</run_depend>
+  <run_depend>rviz</run_depend>
+  <run_depend>xacro</run_depend>
+
+  <export>
+    <architecture_independent/>
+  </export>
+</package>
```

Comment:
Would it not be an idea to submit this to 'upstream' (ie: `ros-industrial/abb_experimental`)?